### ISSUE TITLE
Post Release Task: Updated Stable Tags not propagated to other release branches

### ISSUE THREAD

**Issue by psealock** [woothemes/woocommerce#36136]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

**Issue Description:**

Stable tags and other version bumps are done as part of Code Freeze action

https://github.com/woocommerce/woocommerce/blob/9ee2948954d9025a577f0d402ac4ccc938cb7f31/.github/workflows/release-code-freeze.yml#L106-L108

When a fix release happens, the version bump tool is run manually and a pull request is created. See https://github.com/woocommerce/woocommerce/pull/36108

When the next release branch has already been cut and a fix release happens in a previous release, the stable tag needs to be updated in other ongoing release branches. We have neither automation nor documentation in the release guide. This issue is to evaluate an automated process or if thats too complicated to document steps required.

**Comment by psealock** [comment]
For now I've updated the release lead guide

<img width="831" alt="image" src="https://user-images.githubusercontent.com/1922453/209035824-4b08e052-b75d-43d0-b53d-076f3de10d86.png">

**Comment by roykho** [comment]
I've looked at this and as I remember we've already decided in the past that any fix releases will need manual updates to changelog and stable versions. This is why we have automated checks [here](https://github.com/woocommerce/woocommerce/blob/trunk/.github/workflows/post-release.yml#L53) to see if it is a fix release.

So I think the only thing we need is update the release documentation to state the current release branch will need an update to the stable version as well. I will update the docs and close this.
